src/functTest/groovy/com/bmuschko/gradle/docker/tasks/image/DockerBuildImageFunctionalTest.groovy

```diff
@@ -203,6 +247,7 @@ class DockerBuildImageFunctionalTest extends AbstractGroovyDslFunctionalTest {
 
             task dockerfile(type: Dockerfile) {
                 from '$TEST_IMAGE_WITH_TAG'
+                runCommand('pwd')
```

Comment:
Can you explain why this was necessary?

Reply:
I have to add another layer on top of the alpine image. If I add no layer it uses Alpine:3.4 as is and since this image was used to build some child images, I cannot remove when it's time to test deleting image and check up-to-date status.